pkg/cli/create/user_identity_mapping.go

```diff
@@ -126,7 +125,7 @@ func (o *CreateUserIdentityMappingOptions) Run() error {
 		User:     corev1.ObjectReference{Name: o.User},
 	}
 
-	if err := util.CreateOrUpdateAnnotation(o.CreateSubcommandOptions.CreateAnnotation, mapping, scheme.DefaultJSONEncoder()); err != nil {
+	if err := util.CreateOrUpdateAnnotation(o.CreateSubcommandOptions.CreateAnnotation, mapping, createCmdJSONEncoder()); err != nil {
```

Comment:
If this is possible, I think it is better to use `scheme.DefaultJSONEncoder` for the ones that don't require custom scheme?

Reply:
It should be possible. Do you think it could be a little confusing though? Imagine looking at the directory tree for `pkg/cli/create`, and seeing a `scheme.go` file there. You would instinctively know that this package uses its own scheme instead of relying on the global one. Maybe you need to change something in it, and when you do, it doesn't do what you expected. I don't know if this is a real possibility with the json encoder, but it's worth considering. 
WDYT?

Reply:
yeah, I agree with you. That creates confusion.